.github/workflows/ci-cd.yml

```
@@ -0,0 +1,160 @@
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          args: --severity-threshold=high
+          
+      - name: Run CodeQL Analysis
+        uses: github/codeql-action/analyze@v3
+        with:
+          category: "/language:javascript,/language:typescript"
+          
+      - name: Run Dependency Review
+        uses: actions/dependency-review-action@v3
```

actionlint (1.7.7)

the runner of "actions/dependency-review-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)